chart/kubeapps/Chart.yaml

```diff
@@ -1,19 +1,23 @@
-apiVersion: v1
-name: kubeapps
-version: 4.0.5
+annotations:
+  category: Infrastructure
+apiVersion: v2
 appVersion: DEVEL
+dependencies:
+  - name: postgresql
+    repository: https://charts.bitnami.com/bitnami
+    version: '10.X.X'
```

Comment:
would the major upgrade of PG work out of the box or is there any additional step to run? (I guess it's only the chart v1 deprecation but just to double check).

Reply:
For now yes, we are also evaluating if we should include as part of this PostgreSQL major a rewording of the master/slave nomenclature to something like primary/readReplica; but it is something we're still evaluating. If finally we also apply those changes, I will update the upgrade notes according to that